Zend/zend.c

```diff
@@ -1984,6 +1996,9 @@ void free_estring(char **str_p) /* {{{ */
 }
 /* }}} */
 
+ZEND_API size_t zend_map_ptr_static_size;
+ZEND_API size_t zend_map_ptr_static_last;
```

Comment:
Any reason these are mutable globals as opposed to thread-locals?

Reply:
That's the point - they're supposed to be allocated fully during startup. And startup is the same across all threads.